crates/oxc_transformer/src/react_jsx/mod.rs

```diff
@@ -124,24 +124,29 @@ impl<'a> ReactJsx<'a> {
         }
     }
 
+    fn get_jsx_runtime_importer(&self) -> String {
+        let import_source = self.options.import_source.as_deref().unwrap_or("react");
+        format!("{import_source}/jsx-runtime")
```

Comment:
This is allocating a string for every function call, which hurts performance.
What we should do instead is allocate once and store this into the struct, and then just reference it.

Reply:
I got your meaning. But seems hard to do, There are both mutable borrowing and immutable borrowing here.

Reply:
Can you push what you have tried? I'll take a look.

Reply:
I pushed, Thank you.